src/Core.Assets/esbuild.config.mjs

```diff
@@ -1,5 +1,5 @@
 import * as esbuild from 'esbuild'
-import pkg from './package.json' assert { type: 'json' }
+import pkg from './package.json' with { type: 'json' }
```

Comment:
I had Eslint error's when building and found that "with" is the new/ recommend way.

See the following link:
https://nodejs.org/api/esm.html#data-imports
https://github.com/eslint/eslint/discussions/15305#discussioncomment-8248405

![image](https://github.com/user-attachments/assets/9f50895d-61fd-45a9-9fd2-f551375454ad)

Reply:
Thanks for this info.

Reply:
Visual Studio gives me an error when using the "with" syntax. I already installed Node 22.6.0 and restarted VS. What is the secret to get this to work with "with"?

Reply:
yup, tried all of that. Plus Clean/Rebuild in VS. Still errors.

Reply:
Hmm weird. For my self it was as easy as doing a npm i and then build. But I use mac without visual studio and had to run the command my self in the terminal.

I would like to check for you why visual studio won't build but I'm currently on vacation. Maybe for the time being change it back to "assert" and I Will look in to it after my vacation.

And when do you expect there will be a new release. Because the fix for the pagination with the itemprovider is a production blocker in one of my companies web applications.

Reply:
I'm setting it back indeed for another PR I'm working on,

I myself have just returned from vacation. Need to work on shortening the huge list of open issues that got created during my absence and will do a new release then. Will probably be somewhere in the next two weeks (we never commit to specific release dates)